include/deal.II/fe/fe_tools.h

```diff
@@ -777,6 +782,29 @@ namespace FETools
                     const DoFHandler<dim,spacedim> &dof2,
                     const ConstraintMatrix         &constraints,
                     OutVector                      &z2);
+
+  /**
+   * Same as above for external parallel VectorTypes
+   * on a parallel::distributed::Triangulation
+   */
+  template <int dim, class VectorType, int spacedim>
+  void extrapolate_parallel(const DoFHandler<dim,spacedim> &dof1,
+                            const VectorType &u1,
+                            const DoFHandler<dim,spacedim> &dof2,
+                            const ConstraintMatrix &constraints2,
+                            VectorType &u2);
+
+  /**
+   * Same as above for deal.II parallel Vectors
+   * on a parallel::distributed::Triangulation
+   */
+  template <int dim, int spacedim, typename Number>
+  void extrapolate_parallel(const DoFHandler<dim,spacedim> &dof1,
```

Comment:
I can't specialize that function partially and didn't want to write that code for every possible value of all the template arguments.